dev-libs/libcgroup/libcgroup-0.41-r6.ebuild

```diff
@@ -75,16 +87,9 @@ src_configure() {
 		${my_conf}
 }
 
-src_test() {
-	# Use mount cgroup to build directory
-	# sandbox restricted to trivial build,
-	# possible kill Diego tanderbox ;)
-	true
-}
-
 src_install() {
 	default
-	prune_libtool_files --all
+	find "${D}" -name '*.la' -delete || die
 
 	insinto /etc/cgroup
 	doins samples/*.conf || die
```

Comment:
Also you want to fix those superfluous dies.

Reply:
Fixed :)